sci-visualization/mantid/files/mantid-3.3.0-minigzip-OF.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/MantidPlot/src/zlib123/minigzip.c b/MantidPlot/src/zlib123/minigzip.c
+index 97fbae1..5f4612f 100644
+--- a/MantidPlot/src/zlib123/minigzip.c
++++ b/MantidPlot/src/zlib123/minigzip.c
```

Comment:
No, it's not possible to depend on minizip because this is miniGzip -- it works on .gz files not .zip files. As far as I can tell, this uses system zlib as there is no bundled zlib. (I didn't include sys-libs/zlib in DEPEND because it is part of @system -- is that the right thing to do?)